src/test/ui/issues/issue-45696-scribble-on-boxed-borrow.rs

```diff
@@ -1,28 +1,15 @@
 // rust-lang/rust#45696: This test is checking that we *cannot* return
 // mutable borrows that would be scribbled over by destructors before
 // the return occurs.
-//
-// We will explicitly test NLL, and migration modes;
-// thus we will also skip the automated compare-mode=nll.
```

Comment:
Wait a minute: In tests like this, If you are removing the explicit testing of the distinct revisions, then you should also remove the `// ignore-compare-mode-nll`.

Reply:
Fixed in https://github.com/rust-lang/rust/pull/64221/commits/5a0e4613ce62f1e46f974af4065e5ccb3f56a1af over at https://github.com/rust-lang/rust/pull/64221.